### PR TITLE
Restore broken `release-download-count`

### LINKED ISSUE #5634: `release-download-count`: Cannot read properties of null (reading 'after')
### Please ensure:

- [X] The bug is caused by Refined GitHub. It doesn't happen if I disable the extension.
- [X] The bug happens after clearing extension cache. The "Clear cache" button below can also be found at the bottom of the options page.

### Example URLs

https://github.com/notepad-plus-plus/notepad-plus-plus/releases

### Extension version

22.4.21

### Browser(s) used

Chrome

### Description

Probably because of https://github.blog/changelog/2022-05-18-creation-dates-for-release-assets/

```
TypeError: Cannot read properties of null (reading 'after')
    at init (chrome-extension://hlepfoohegkhhmjieoechaddaejaokhf/refined-github.js:8293:291)
    at async runFeature (chrome-extension://hlepfoohegkhhmjieoechaddaejaokhf/refined-github.js:3902:20)
    at async setupPageLoad (chrome-extension://hlepfoohegkhhmjieoechaddaejaokhf/refined-github.js:3910:34)
```

### Include in this issue:

- [ ] Screenshots/video/gif demonstrating the bug, if it’s visual
- [X] Console errors, if any

### PR DESCRIPTION
**Pull request by fregante**
- Fixes #5634 

## Test URLs

https://github.com/notepad-plus-plus/notepad-plus-plus/releases

## Screenshot

Screen Shot

### CODE REVIEW

File: source/features/release-download-count.tsx [hunk 1]
```diff
@@ -78,12 +78,23 @@ async function init(): Promise<void | false> {
...
+					'rgh-release-download-count',
+					...assetSize.classList,
+				].map(name => name.replace('text-sm-left', 'text-sm-right'));
+
+				if (index === 0) {
+					classes.push('text-bold');
+				}
+
+				assetSize
 					.after(
```
Comment (cheap-glitch) [inline source/features/release-download-count.tsx:95]:
```suggestion
				assetSize.after(
```

File: source/features/release-download-count.tsx [hunk 2]
```diff
@@ -78,12 +78,23 @@ async function init(): Promise<void | false> {
...
-				assetName
+				const assetSize = assetName
 					.closest('.Box-row')!
-					.querySelector('.float-right')!
+					.querySelector(':scope > .flex-justify-end > :first-child')!;
+
+				const classes = [
+					'rgh-release-download-count',
+					...assetSize.classList,
+				].map(name => name.replace('text-sm-left', 'text-sm-right'));
```
Comment (cheap-glitch) [inline source/features/release-download-count.tsx:88]:
```suggestion
				const classes = [
					'rgh-release-download-count',
					...assetSize.classList,
				];
```

File: source/features/release-download-count.tsx [hunk 3]
```diff
@@ -78,12 +78,23 @@ async function init(): Promise<void | false> {
...
+
+				if (index === 0) {
+					classes.push('text-bold');
+				}
+
+				assetSize
 					.after(
 						<small
-							className={'rgh-release-download-count float-right mr-2 color-text-secondary color-fg-muted' + (index === 0 ? ' text-bold' : '')}
+							className={classes.join(' ')}
```
Comment (cheap-glitch) [inline source/features/release-download-count.tsx:97]:
```suggestion
							className={classes.join(' ').replace('text-sm-left', 'text-sm-right')}
```

**Review by cheap-glitch — APPROVED** [review]
Just some lint suggestions

File: source/features/release-download-count.tsx (hunk 1 above)
Comment (fregante) [reply source/features/release-download-count.tsx:95]: Prettier made me forget I need to handle this stuff manually here 🥲

**Pull request merged**